app/assets/stylesheets/admin/views/_corporate-information-index.scss

```diff
@@ -0,0 +1,42 @@
+.govuk-tag--s {
+  @include govuk-font(14, $weight: bold, $line-height: 1);
+}
+
+.app-view-corporate-information-index__pagination {
+  ul {
+    padding-inline-start: 0;
+    margin-top: govuk-spacing(8);
+    margin-bottom: govuk-spacing(8);
+  }
+
+  li {
+    display: inline;
+    box-sizing: border-box;
+    border-color: $govuk-border-colour;
+    border: 1px solid;
+    padding: govuk-spacing(2) govuk-spacing(3);
+  }
+
+  li.active {
+    background-color: govuk-colour("blue");
+    border-color: govuk-colour("blue");
+
+    a {
+      color: govuk-colour("white");
+    }
+  }
+
+  li.disabled a {
+    color: govuk-colour("dark-grey");
+    text-decoration: none;
+    cursor: not-allowed;
+  }
+
+  a {
+    color: $govuk-link-colour;
+  }
+
+  a:hover {
+    color: $govuk-link-hover-colour;
+  }
```

Comment:
I imagine there's a far better way to import govuk-link behaviour than this. The ways i found complained in the linter though. V open to suggestions!